ui/src/Error.elm

```diff
@@ -0,0 +1,28 @@
+-- Copyright (c) 2019
+-- Mainflux
+--
+-- SPDX-License-Identifier: Apache-2.0
+
+
+module Error exposing (handle)
+
+import Http
+
+
+handle : Http.Error -> String
+handle error =
+    case error of
+        Http.BadUrl url ->
+            "Bad url: " ++ url
```

Comment:
I prefer `Bad URL`

Reply:
@manuio This one's kind of temporary... Will be refactored.

Reply:
@darkodraskovic I would understand if PR was ready to merge but you have to fix `Disonnect`. You can at the same time fix `ID` in all table and `Bad URL` here.